acrg/paris_formatting/make_paris_outputs.py

```diff
@@ -115,24 +115,33 @@ def make_country_output(
         "NW_EU_CONTINENT": "BEL-DEU-FRA-LUX-NLD",
     }
 
-    region_traces = []
-    for region, countries_str in regions_dict.items():
-        region_countries = countries_str.split("-")
-        region_ds = (
-            country_traces_merged.sel(country=region_countries).sum("country").expand_dims({"country": [region]})
+    try:
+        region_traces = []
+        for region, countries_str in regions_dict.items():
+            region_countries = countries_str.split("-")
+            region_ds = (
+                country_traces_merged.sel(country=region_countries).sum("country").expand_dims({"country": [region]})
+            )
+            region_traces.append(region_ds)
+
```

Comment:
You can catch the exception here, since by this point, we should get an error if a country is missing.